src/pybind/mgr/dashboard/frontend/src/testing/unit-test-helper.ts

```diff
@@ -101,3 +104,60 @@ export class PermissionHelper {
     });
   }
 }
+
+export class FormHelper {
+  form: CdFormGroup;
+
+  constructor(form: CdFormGroup) {
+    this.form = form;
+  }
+
+  private getControl(control: AbstractControl | string): AbstractControl {
+    if (typeof control === 'string') {
+      return this.form.get(control);
+    }
+    return control;
+  }
+
+  hasError(control: AbstractControl | string, error: string) {
+    expect(this.getControl(control).hasError(error)).toBeTruthy();
+  }
+
+  isValid(control: AbstractControl | string) {
```

Comment:
`isValid` sounds like it returns a boolean value, but it runs an `expect` internally. What would you think of `expectValid` instead? The same applies to other methods which expect something, like `hasError` -> `expectError`.

Reply:
Expect is a bit to gross, `has` will give the context that we are searching for a specific error not just some error. `is` is used for methods that can only be one state or another (boolean).
Every method that has an expect in it starts with `is`/`are` or `has`. Calling all `expect` would remove some context.

Reply:
Per our discussion during today's standup, the concensus was to change this to `expect` instead of `is`.